lambdas/list_tiled_analysis/src/lambda_function.py

```diff
@@ -0,0 +1,49 @@
+from aws_xray_sdk.core import patch, xray_recorder
+from shapely.geometry import mapping
+from shapely.wkb import loads
+
+from raster_analysis.data_environment import DataEnvironment
+from raster_analysis.exceptions import QueryParseException
+from raster_analysis.globals import LOGGER
+from raster_analysis.tiling import AnalysisTiler
+
+patch(["boto3"])
+
+
+@xray_recorder.capture("Tiled Analysis")
+def handler(event, context):
```

Comment:
This is because it's a different input, right? Would it be possible to pull out the common code into a function and have the handlers just pull out the parameters from the input?

Reply:
I think you're talking about the fact that we that the list_tiled_analysis lambda and the tiled_analysis lambda and they are fairly similar?  As you say, it is because the inputs are slightly different (we need to decode the WKB, we need to deal with the fid, no format arg, etc.)  Most of the common code is already shared in the calls to AnalysisTiler() and tiler.execute().  I'll take a look, but I'm not sure if it is really worth sharing any more code, given the remaining code is mostly arg and results handling.